### PR TITLE
[rjsf] Styles fixes 

### PR DESCRIPTION
**Pull request by harkiratsm**
Signed-off-by: harkiratsm <multaniharry714@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**
Two Columns Layout.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: ui/themes/rjsf.js [hunk 1]
```diff
@@ -38,13 +38,21 @@ export const rjsfTheme = createTheme({
     MuiInputLabel : {
       root : {
         whiteSpace : "nowrap",
-        // overflow : "hidden",
+        overflow : "hidden",
```
Comment (leecalcote) [inline ui/themes/rjsf.js:41]: Good update. 👍

File: ui/themes/rjsf.js [hunk 2]
```diff
@@ -38,13 +38,21 @@ export const rjsfTheme = createTheme({
...
         textOverflow : "ellipsis",
         maxWidth : "75%",
         height : "100%",
         '&:hover' : {
           overflow : "visible",
         }
+      },
+      shrink : {
+        maxWidth : "133%",
```
Comment (leecalcote) [inline ui/themes/rjsf.js:50]: Good update

File: ui/themes/rjsf.js [hunk 3]
```diff
@@ -38,13 +38,21 @@ export const rjsfTheme = createTheme({
...
+      }
+    },
+    MuiFormControl : {
+      root : {
+        marginTop : "0.1em"
```
Comment (leecalcote) [inline ui/themes/rjsf.js:55]: Hmm. I don't know that this is desirable.

**Comment by harkiratsm** [comment]
Newer Changes - 
1) Two Columns Layout for the Design Configurator 
& fix three items from this https://app.clickup.com/t/2wnamcr

File: ui/themes/rjsf.js [hunk 4]
```diff
@@ -152,6 +160,14 @@ export const rjsfTheme = createTheme({
           whiteSpace : "normal",
         }
       },
+      "grid-xs-6" : {
+        paddingRight : '0.5rem'
+      }
+    },
+    MuiIconButton : {
+      root : {
+        cursor : "default"
```
Comment (leecalcote) [inline ui/themes/rjsf.js:169]: Good.

File: ui/themes/rjsf.js [hunk 5]
```diff
@@ -32,19 +32,27 @@ export const rjsfTheme = createTheme({
...
       }
     },
     MuiInputLabel : {
       root : {
         whiteSpace : "nowrap",
-        // overflow : "hidden",
+        overflow : "hidden",
         textOverflow : "ellipsis",
         maxWidth : "75%",
         height : "100%",
```
Comment (leecalcote) [inline ui/themes/rjsf.js:44]: 100% = good.

File: ui/themes/rjsf.js [hunk 6]
```diff
@@ -152,6 +160,14 @@ export const rjsfTheme = createTheme({
           whiteSpace : "normal",
         }
       },
+      "grid-xs-6" : {
+        paddingRight : '0.5rem'
```
Comment (leecalcote) [inline ui/themes/rjsf.js:164]: display: flex; justify-content: space-around;

**Comment by harkiratsm** [comment]
This PR fixes - 
1) Fixed the behaviour when input field .
2) On expand the items/object/array fields are not expanding.
3) Be consistent with cursor on tooltip - default pointer (arrow).
4) Apply a bottom margin to name and namespace + no overlapping.
5) Fixed labels and annotations behaviour in Design Configurator .
...... typing

**Comment by leecalcote** [comment]
@harkiratsm signal after today's feedback has been incorporated.

**Comment by harkiratsm** [comment]
Newer Changes are - scaling the fields using ```space-around``` .

**Comment by harkiratsm** [comment]
@leecalcote

**Comment by leecalcote** [comment]
@harkiratsm provide a revised, itemized list of changes, please.

**Comment by harkiratsm** [comment]
> @harkiratsm provide a revised, itemized list of changes, please.

https://github.com/meshery/meshery/pull/6360#issuecomment-1277436451

File: ui/themes/rjsf.js [hunk 7]
```diff
@@ -9,6 +9,7 @@ export const rjsfTheme = createTheme({
     },
   },
   typography : {
+    fontFamily : "Qanelas Soft, sans-serif",
```
Comment (leecalcote) [inline ui/themes/rjsf.js:12]:
```suggestion
```

**Comment by leecalcote** [comment]
@harkiratsm I don't know what this means: `[ Not accepted currently ]`

**Pull request merged**